README.md

```diff
@@ -374,10 +374,6 @@ Various queue related functionality has been added to the template project so th
 
 Note that this module does not produce an artifact for publishing - we only publish the library from the `hmpps-sqs-spring-boot-starter` module.
 
-#### test-app-async
-
-This is a copy of the `test-app` which uses a Reactive Spring configuration. This is required as more and more projects switch to using Reactive in their applications. It should be kept up-to-date with `test-app`. All following instructions relating to the test-app apply to test-app-async too.
-
```

Comment:
I don't think we should delete the test-app-async. It's there to test the reactive API where everything is a suspend function which was written for a project that is fully asynchronous. To be honest it's badly named - should be `test-app-reactive`.